webapp/src/domain/schemas/AllSchema.tsx

```diff
@@ -14,15 +14,38 @@ import ActionMenu, { useShareLink, useChangeAccess, useDelete } from "../../comp
 import ButtonLink from "../../components/ButtonLink"
 import { CellProps, Column } from "react-table"
 import { SchemaDispatch } from "./reducers"
-import { Access, Schema } from "../../api"
+import Api, { Access, Schema,} from "../../api"
+import { alertAction } from "../../alerts"
 import SchemaImportButton from "./SchemaImportButton"
-
+ 
 type C = CellProps<Schema>
 
 export default function AllSchema() {
     document.title = "Schemas | Horreum"
     const dispatch = useDispatch<SchemaDispatch>()
 
+    const [page, setPage] = useState(1)
+    const [perPage, setPerPage] = useState(20)
+     
+    const pagination:any = useMemo(() => ({ page, perPage}), [page, perPage])
+     
+    const [schemas, setSchemas] = useState<Array<Schema>>();
+    const [loading, setLoading] = useState(false)
+
+     useEffect(() => {
+        setLoading(true)
+        Api.schemaServiceList(
+
```

Comment:
we need to add the SortDirection here, otherwise the param mapping for the api call is incorrect, something like;

```
        Api.schemaServiceList(
            SortDirection.Ascending,
            pagination.perPage,
            pagination.page,
```